public/app/plugins/datasource/testdata/components/SimulationQueryEditor.tsx

```diff
@@ -79,6 +119,12 @@ export const SimulationQueryEditor = ({ onChange, query }: EditorProps) => {
           <Input type="text" placeholder="optional" value={simQuery.key.uid} onChange={onUIDChanged} />
         </InlineField>
       </InlineFieldRow>
+      <div>
+        CONFIG:
+        <pre>{JSON.stringify(config.value, null, 2)}</pre>
```

Comment:
@lean257 -- can you look at chaning the above line to a form you can submit with `ds.postResource(...)`

That seems like the easiest way to start building a form